_projects/git_task_list.md

```diff
@@ -0,0 +1,82 @@
+---
+collaborating_projects:
+  - gsoc-prep-tasks
+desc: "List pull request/merge request (in GitLab) that have no reviewer yet. Provide an easy way for new comer / developer to do contribution to a project by giving review to other people pull requests."
+difficulty: medium
+initiatives:
+  - GSoC
+issues:
+  - https://github.com/coala/projects/issues/480
+  - https://github.com/coala/coala/issues/3615
+  - https://github.com/coala/coala/issues/4080
+markdown: git_task_list.md
+mentors:
+  - sims1253
+  - yukiisbored
+  - yashovardhanagrawal
+name: "Git Task List"
+requirements:
+  - "The applicant should have at least one patch merged to any of the coala repositories."
+  - "Familiarity with Git, GitHub API Gerrit API, Phabricator API, and GitLab API"
+  - "Familiarity with EmberJS, Javascript, and GitHub Pages"
+status: in_progress
+tags:
+  - community
+  - web
+---
+The basic idea is an extended version of [GSOC Task List](http://summerofcode.github.io/gsoc-prep-tasks)
+which was initialy intended for GSOC Applicans who don't have or only have a
+litle experience in open-source. Hence, [that](http://github.com/summerofcode/gsoc-prep-tasks)
+project will be the starting point of this project.
+
+This project will help sharing the tasks of reviewing a pull request on a
+project. It will help fix the problem where developer spend too much time
+finding PR that haven’t been reviewed. This projects aims to getting more
+developer doing reviews works and sharing reviews workload, by providing an easy
+way to find issues that has not been reviewed. It also help project’s maintainer
+/ developer do review more efficient.
+
+This also help engage developer to keep contributing to a project where they
+can’t find any issue that they can fix or keep the new comer issue for new comer
+as learning path. And developer with more experience can contribute by reviewing
+new comer’s work PR
+
+This project is also related to “New Comer Metrics And Gamification”. Where some
+of the activities that will be given some point is involving:
+
+- Issues
+  - Comment to an Issue
+  - Create new Issue
+- Pull request
+  - Review to a Pull Request
+  - Create a Pull request
+
+Which is addressed by this project.
+
+The Goal of this projects is a serverless web page where the user can find Pull
+Request that haven’t been reviewed. User can select projects in Open Source
+hosting services like GitHub or GitLab. They also have the ability to select
```

Comment:
Are other projects hosting sites outside of GitHub/GitLab are considered or is it just github and gitlab for now?